README.md

```diff
@@ -1,2 +1,88 @@
-# bitstring
-A Bitstring module for universal JavaScript
+# Bitstring
+A Bitstring module for universal JavaScript.
+
+## Install
+
+To install get the latest version from NPM
+
+```
+npm install @digitalbazaar/bitstring
+```
+
+To install locally for development:
+
+```
+git clone https://github.com/digitalbazaar/bitstring.git
+cd bitstring
+npm install
+```
+
+## Usage
+
+```js
+import Bitstring from '@digitalbazaar/bitstring';
+
+// You can make a new instance by passing in a length
+const bitstring = new Bitstring({length: 10});
+
+// OR by passing in a buffer
+const buffer = Uint8Array.from([255]);
+const bitstring = new Bitstring({buffer});
+```
+
+## API
+
+### set(position, on)
+Sets the value of a bit for the given position to the given boolean.
+```js
+bitstring.set(1, true);
+```
+
+### get(position)
+Gets the value of a bit for the given position.
+```js
+bitstring.get(3);
+```
+
+### encodeBits()
+Compresses and encodes bits to a base64url encoded string.
+
+```js
+await bitstring.encodeBits();
+```
+
+### decodeBits({encoded})
+Decodes and uncompresses a given a base64url encoded string to a Uint8Array.
+```js
+await Bitstring.decodeBits({encoded});
+```
+
+### compressBits()
+Compresses bits to a Uint8Array.
+```js
+await bitstring.compressBits();
```

Comment:
I might be wrong here, but it looks like compressBits gzips a Uint8array:

https://github.com/digitalbazaar/bitstring/blob/b11fb56cecb96cd00b097ef709726f65423bcef0/main.js#L55-L57

Reply:
@aljones15 yeah, you're right `this.bits` is Uint8Array but the compressed output is also Uint8Array. So, I think this is fine.

Reply:
ok did not know the gzip operation produced a Uint8Array, but that does makes sense. Don't let this comment hold you up feel to merge as you have multiple approvals.